### DISCUSSION TITLE
Showing Infinite loading

### DISCUSSION THREAD

**Discussion by saurabhnextbits** [directus/directus#17430] [Archive]
### Describe the Bug

Got in finite loading screen till approx 6pm IST 
and after that project loading normally and login page visible

### To Reproduce

local setup with directus repo
running 
pnpm --filter directus dev  in 1st terminal
pnpm --filter @directus/app dev in 2nd terminal

### What version of Directus are you using?

current

### What version of Node.js are you using?

18.0

### What database are you using?

don't know

### What browser are you using?

chrome

### How are you deploying Directus?

 running locally

**Comment by fabiobeoni** [comment]
Same here...

Two terminals open as described in the documentation, one for api and one for the app:

**App:**

![Screenshot from 2023-03-08 22-13-23](https://user-images.githubusercontent.com/16882089/223851484-cdca3aef-ebd7-42a8-8d99-ecd6651a90e3.png)

**API:**

![Screenshot from 2023-03-08 22-14-27](https://user-images.githubusercontent.com/16882089/223851602-e833ffc3-424a-41e2-a677-6fbbf8979c0b.png)

UI never comes to life...

![Screenshot from 2023-03-08 22-15-51](https://user-images.githubusercontent.com/16882089/223851953-4b285408-39fb-47cd-9eb0-909ed4e12f95.png)

780 http requests.... all completed with 200 or 304 status code but the web page keeps loading....

I restarted build and run of API/APP several times..

**Reply by fabiobeoni** [reply]
In the JS console I see:

```
[vite] connecting...
[vite] connected.
```
But again, white web page.
